app/Listeners/UpdateReplyableActivity.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Listeners;
+
+use App\Events\ReplyWasCreated;
+
+final class UpdateReplyableActivity
```

Comment:
```suggestion
final class MarkLastReplyMoment
```

Reply:
In light of the column name, let's name this:

```suggestion
final class MarkLastActivity
```